src/main/java/com/spikes2212/dashboard/Namespace.java

```diff
@@ -10,45 +15,217 @@
+     * @return the desired value
+     */
     String getString(String key);
 
+    /**
+     * Adds a number supplier to the namespace.
+     *
+     * @param key   the key that will be given to the value
+     * @param value the value to be added
+     */
     void putNumber(String key, Supplier<? extends Number> value);
 
+    /**
+     * Adds a number value to the namespace.
+     *
+     * @param key    the key that will be given to the value
+     * @param number the value to be added
+     */
     default void putNumber(String key, Number number) {
         this.putNumber(key, () -> number);
     }
 
+    /**
+     * Gets a number value from the namespace.
+     *
+     * @param key the key of the value
+     * @return the desired value
+     */
     double getNumber(String key);
 
+
+    /**
+     * Adds a boolean supplier to the namespace.
+     *
+     * @param key   the key that will be given to the value
+     * @param value the value to be added
+     */
     void putBoolean(String key, Supplier<Boolean> value);
 
+
+    /**
+     * Adds a boolean value to the namespace.
+     *
+     * @param key   the key that will be given to the value
+     * @param value the value to be added
+     */
     default void putBoolean(String key, boolean value) {
         this.putBoolean(key, () -> value);
     }
 
+    /**
+     * Gets a boolean value from the namespace.
+     *
+     * @param key the key of the value
+     * @return the desired value
+     */
     boolean getBoolean(String key);
 
+    /**
+     * Adds a set of {@link PIDSettings} values to the namespace to a designated {@link ChildNamespace}.
+     *
+     * @param name               the name to be given to the settings and the child namespace
+     * @param initialPIDSettings the initial values for the PID settings to be added
+     * @return PID settings with the values from the network tables
+     */
+    default PIDSettings addPIDNamespace(String name, PIDSettings initialPIDSettings) {
+        ChildNamespace child = this.addChild(name + " pid");
+        Supplier<Double> kP = child.addConstantDouble("kP " + name, initialPIDSettings.getkP());
+        Supplier<Double> kI = child.addConstantDouble("kI " + name, initialPIDSettings.getkI());
+        Supplier<Double> kD = child.addConstantDouble("kD " + name, initialPIDSettings.getkD());
+        Supplier<Double> tolerance = child.addConstantDouble(name + " tolerance", initialPIDSettings.getTolerance());
+        Supplier<Double> waitTime = child.addConstantDouble(name + " wait time", initialPIDSettings.getWaitTime());
+        return new PIDSettings(kP, kI, kD, tolerance, waitTime);
+    }
+
+    /**
+     * Adds a set of {@link FeedForwardSettings} values to the namespace to a designated {@link ChildNamespace}.
+     *
+     * @param name                       the name to be given to the settings and the child namespace
+     * @param initialFeedForwardSettings the feed forward settings to be added
```

Comment:
specify that these are the initial values only, similarly to how it was done with the PIDSettings